src/components/MainPanel/ModeDevicePopover.scss

```
@@ -0,0 +1,199 @@
+// pattern. Each row has a dedicated mute toggle button.
+.mode-device-popover {
+  background: #2a2a2a;
+  border: 1px solid #444;
+  border-radius: 8px;
+  width: 320px;
+  font-size: 12px;
+  box-shadow: 0 4px 16px rgba(0, 0, 0, 0.5);
+  color: #ccc;
+  z-index: 1000;
+  overflow: hidden;
+  // max-height is set inline by floating-ui's size() middleware.
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)